src/initialization.js

```diff
@@ -1,42 +1,99 @@
-        /* `forwarderSettings` contains your SDK specific settings such as apiKey that your customer needs in order to initialize your SDK properly */
-
+    initForwarder: function (
+        forwarderSettings,
+        testMode,
+        _userAttributes,
+        _userIdentities,
+        processEvent,
+        eventQueue,
+        _isInitialized,
+        _common,
+        _appVersion,
+        _appName,
+        _customFlags,
+        _clientId
+    ) {
         if (!testMode) {
-            /* Load your Web SDK here using a variant of your snippet from your readme that your customers would generally put into their <head> tags
-               Generally, our integrations create script tags and append them to the <head>. Please follow the following format as a guide:
-            */
+            if (!window.Rokt || !(window.Rokt && window.Rokt.currentLauncher)) {
+                var target = document.head || document.body;
+                var script = document.createElement('script');
+                script.type = 'text/javascript';
+                script.src = roktLauncherScript;
+                script.async = true;
+                script.crossOrigin = 'anonymous';
+                script.fetchPriority = 'high';
+                script.id = 'rokt-launcher';
+
+                script.onload = function () {
+                    // Once the script loads, ensure the Rokt object is available
+                    if (
+                        window.Rokt &&
+                        typeof window.Rokt.createLauncher === 'function' &&
+                        window.Rokt.currentLauncher === undefined
+                    ) {
+                        window.Rokt.createLauncher({
+                            accountId: forwarderSettings.accountId,
+                            sandbox:
+                                forwarderSettings.sandboxMode &&
+                                forwarderSettings.sandboxMode === 'True'
+                                    ? true
+                                    : false,
+                        })
+                            .then(function (launcher) {
+                                // Assign the launcher to a global variable for later access
+                                window.Rokt.currentLauncher = launcher;
+                                if (window['Rokt'] && eventQueue.length > 0) {
+                                    for (
+                                        var i = 0;
+                                        i < eventQueue.length;
+                                        i++
+                                    ) {
+                                        processEvent(eventQueue[i]);
+                                    }
+                                    eventQueue = [];
+                                }
```

Comment:
Is there a use case for an mparticle event to be passed to the Rokt SDK for processing? I don't think so.  It doesn't hurt to have this, but if Rokt never processes mParticle events, then this can be removed.
```suggestion
```